src/components/VBackToSearchResultsLink.vue

```diff
@@ -5,12 +5,14 @@
     v-bind="$attrs"
```

Comment:
Maybe rename this component to `VBackLink`?

Reply:
We can remove this change entirely now, right? The component is no longer used in the new report page.